src/python/espressomd/interactions.pxd

```diff
@@ -463,14 +463,13 @@ cdef extern from "interaction_data.hpp":
         IBM_Tribend_Parameters ibm_tribend
         IBM_VolCons_Parameters ibmVolConsParameters
 
-    ctypedef struct bonded_ia_parameters:
+    cdef struct Bonded_ia_parameters:
```

Comment:
Shouldn't this then be `Bonded_IA_parameters`, for consistencie's sake? I get that you just want to capitalize the first letter, however I am not entirely happy with the multitude of `B/bond(ed)*ia/IA*param(eters)` variables. Not saying that it was much better before...

Reply:
i have not renamed it, i just used the core name here